.github/workflows/dockerhub.yml

```diff
@@ -14,3 +14,5 @@ jobs:
     secrets:
       DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
       DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+    with:
+      runner: ubuntu-latest-16-core
```

Comment:
missing s at the end
ubuntu-latest-16-cores

Reply:
code catch